scripts/issue-tagging.sh

```diff
@@ -38,7 +38,7 @@ PRs=$(git log --pretty=oneline "$BASE_TAG"..."$LATEST_TAG" | grep 'Merge pull re
 EXIT_CODE=0
 ISSUES=()
 for pr in $PRs; do
-    id=$($GHCLI_BIN pr view "$pr" --json body | grep -oE '(Related issues \| )(.*)?[0-9]+(.*|\r|\n)?(\|)' | sed 's/[^[:digit:]]//g' | sed -z 's/\n//g' || true)
+    id=$($GHCLI_BIN pr view "$pr" --json body | grep -oE '(Related issues \| )(.*)?#[0-9]+(.*|\r|\n)?(\|)' | sed 's/[^[:digit:]]//g' | sed -z 's/\n//g' || true)
```

Comment:
We used to match github issues that way, but it turns out sometimes we put the complete URL instead of just the ID. To match both we don't look for the `#`. I'd suggest we just don't make the script fail when the parsed ID isn't a Github issue. WDYT?